packages/core/contracts/optimistic-assertor/implementation/examples/Insurance.sol

```diff
@@ -0,0 +1,97 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../../interfaces/OptimisticAssertorInterface.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract Insurance {
```

Comment:
Please add short description of the contract. I understand it allows issuing one unlimited time policy per event/payout recipient. There is no limit to making payout requests, but only the first asserted request will settle the insurance payment while OA will settle bonds for all requestors.

Reply:
Yes, I couldn't have said it better myself, thank you!

Reply:
Though this now contradicts with the implementation below where block timestamp is added when issuing insurance - this effectively allows issuing new policy for the same event/payout recipient at each next block timestamp.